plone/app/upgrade/v51/betas.py

```diff
@@ -124,15 +123,24 @@ def update_social_media_fields(context):
     field values as they are now declared as ASCIILine instead of
     TextLine.
     """
+    twitter_username = ''
+    facebook_app_id = ''
+    facebook_username = ''
     from Products.CMFPlone.interfaces.controlpanel import ISocialMediaSchema
     registry = getUtility(IRegistry)
     settings = registry.forInterface(ISocialMediaSchema, prefix='plone')
     if settings.twitter_username:
-        settings.twitter_username = str(settings.twitter_username)
+        twitter_username = settings.twitter_username.encode('ascii', 'ignore')
```

Comment:
as mentioned earlier, using a simple `str()` cast is the only thing we need here as those values would never be unicode, in the first place.